remote-messaging/remote-messaging-store/src/androidTest/java/com/duckduckgo/remote/messaging/store/LocalRemoteMessagingConfigRepositoryTest.kt

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2022 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.remote.messaging.store
+
+import androidx.room.Room
+import androidx.test.platform.app.InstrumentationRegistry
+import com.jakewharton.threetenabp.AndroidThreeTen
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertTrue
+import org.junit.Before
+import org.junit.Test
+import org.threeten.bp.LocalDateTime
+
+class LocalRemoteMessagingConfigRepositoryTest {
```

Comment:
Does this need to be an android test? It seems a bit overkill for just two tests that are testing the logic in the expired method so you don't really need an actual db for that.

Reply:
Since we are making those methods extensions functions of `RemoteMessagingConfig`, we can move them into jvm. No dao involved anymore.